packages/lit-dev-content/site/docs/v3/data/context.md

```diff
@@ -503,6 +503,10 @@ The `context-request` bubbles and is composed.
 
     The context object this event is requesting a value for
 
+- `readonly contextTarget: Element`
+
+    The original context target of the requester
```

Comment:
Personally not a fan of using `context target` to describe `contextTarget`. What about something like:

> The DOM element that initiated the context request

Reply:
I will open a PR on @lit/context to update the param description there so this won't get overwritten when the API docs are automatically generated.